packages/commonwealth/client/scripts/views/pages/CommunityManagement/Contests/ManageContest/steps/DetailsFormStep/PayoutRow/PayoutRow.tsx

```diff
@@ -0,0 +1,98 @@
+import moment from 'moment/moment';
+import React from 'react';
+
+import NumberSelector from 'views/components/NumberSelector';
+import { CWText } from 'views/components/component_kit/cw_text';
+import { MessageRow } from 'views/components/component_kit/new_designs/CWTextInput/MessageRow';
+
+import { getPrizeColor } from '../utils';
+
+import './PayoutRow.scss';
+
+interface PayoutRowProps {
+  payoutStructure: number[];
+  onSetPayoutStructure: (payoutStructure: number[]) => void;
+  index: number;
+  payoutNumber: number;
+}
+
+const PayoutRow = ({
+  payoutStructure,
+  onSetPayoutStructure,
+  index,
+  payoutNumber,
+}: PayoutRowProps) => {
+  const handleInput = (e) => {
+    let value = e.target.value;
```

Comment:
i think we should only allow numbers and %?

Reply:
% character can't be modified/added/deleted in this input. It is added as a visual indicator (per design). Further in this handler, there is logic that handles what kind of character was typed. If its non a number, then there is an early return etc. It should work fine 
![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/71be2b16-8315-4d28-9549-8f5d30bebbf8)

Reply:
ah I see, thanks for the explanation.